lib/utils.js

```diff
@@ -155,7 +178,7 @@ export function compactItem(compareWith: Layout, l: LayoutItem, compactType: Com
     if (compactH) {
       l.x = collides.x + collides.w;
     } else {
-      l.y = collides.y + collides.h;
+      resolveCompactionCollisionByMovingItemDown(l, collides.y + collides.h, layout)
```

Comment:
I think we'll have to create a similar function for horizontal compaction, yeah?